tests/Endpoints/PaymentSources/CreditCardTest.php

```diff
@@ -22,6 +22,7 @@ public function testCreateCreditCardToken(): void
     {
         $creditCard = new CreditCard(
             [
+                'cvc' => '123',
```

Comment:
No test to see if this data can be retrieved?

Reply:
we do not retrieve cvc, only send

Reply:
We are not returning/retrieving `cvc`, it is only outbound

Reply:
Right, but when a CC entity is created, is it not possible to see if the `$creditCard->getCvc()` returns the right thing?